docs/graphs/example-graphs.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Example Graphs
-sidebar_position: 70
+sidebar_position: 30
 ---
 
-GDN  comes with a set of easily graspable graphs that are used to demonstrate the APIs.
+The GDN comes with a set of easily graspable graphs that are used to demonstrate the APIs.
```

Comment:
```suggestion
Macrometa comes with a set of easily graspable graphs that are used to demonstrate the APIs.
```

Reply:
We are demonstrating the APIs? This is not the PR for a major rework, but damn, we need to rework this topic.

Reply:
@diana-macrometa I thought the content in those pages seemed like duplicates of other content and removed them. I also 'rebranded' the Graph Query page as a getting started.

Reply:
Please put the tasks back for now.

Reply:
I added the tasks back